install/scripts/alpaca.sh

```diff
@@ -8,11 +8,5 @@ if [ -f "$HOME_DIR/islandora/install/configs/variables" ]; then
   . "$HOME_DIR"/islandora/install/configs/variables
 fi
 
-cd "$HOME_DIR"
-git clone https://github.com/Islandora-CLAW/Alpaca.git
-cd Alpaca
```

Comment:
Would require a webhook to auto-update.  I guess I'll just live with mangling the install script everytime I have to test un-merged changes...

I'm moving it back to point to head.  Please let's just get this merged.  I've got tons of follow up to this.